ai2thor/controller.py

```diff
@@ -569,6 +596,7 @@ def key_sort_func(scene_name):
             )
 
         self.server.send(dict(action="Reset", sceneName=scene, sequenceId=0))
+        self.last_scene = scene
```

Comment:
what is the purpose of last_scene?

Reply:
If a scene is not passed into reset, then `self.last_scene` is used. This just makes the defaults consistent with `init_params` (see: https://github.com/allenai/ai2thor/pull/605#discussion_r574868354).

Reply:
Oh, wait. I suppose this is a duplicate of `self.scene`, which does the same thing.

Reply:
Fixed in https://github.com/allenai/ai2thor/pull/605/commits/764cd71b4e2e3540049d19806d50b5d087654737